src/connection-manager/manifests.ts

```diff
@@ -162,41 +164,18 @@ export async function updateUnls(): Promise<void> {
         .whereIn('master_key', subquery)
         .orderBy(['master_key', { column: 'seq', order: 'desc' }])
         .then(async (res) => {
-          return res.map((idx: { signing_key: string }) => {
-            return idx.signing_key
-          })
+          return res.map((idx: { signing_key: string }) => idx.signing_key)
         })
 
-      // eslint-disable-next-line max-depth -- necessary depth
-      if (name === 'vl_main') {
-        await query('validators')
-          .whereIn('signing_key', keys)
-          .update({ unl: config.vl_main })
-        await query('validators')
-          .whereNotIn('signing_key', keys)
-          .where('unl', '=', config.vl_main)
-          .update({ unl: null })
-      }
-      // eslint-disable-next-line max-depth -- necessary depth
-      if (name === 'vl_test') {
-        await query('validators')
-          .whereIn('signing_key', keys)
-          .update({ unl: config.vl_test })
-        await query('validators')
-          .whereNotIn('signing_key', keys)
-          .where('unl', '=', config.vl_test)
-          .update({ unl: null })
-      }
-      // eslint-disable-next-line max-depth -- necessary depth
-      if (name === 'vl_dev') {
-        await query('validators')
-          .whereIn('signing_key', keys)
-          .update({ unl: config.vl_dev })
-        await query('validators')
-          .whereNotIn('signing_key', keys)
-          .where('unl', '=', config.vl_dev)
-          .update({ unl: null })
-      }
+      const network = networks.filter((ntwk) => ntwk.network === name)[0]
+      const networkUNL = network.unls[0]
```

Comment:
Going from networks -> networkUNL is done twice like this, maybe it's worth making a helper function?